cmd/juju/cloud/list_test.go

```diff
@@ -39,6 +50,34 @@ func (s *listSuite) TestListPublic(c *gc.C) {
 	c.Assert(out, gc.Matches, `.*localhost[ ]*1[ ]*localhost[ ]*lxd.*`)
 }
 
+func (s *listSuite) TestListController(c *gc.C) {
+	// need to add controller details to the store?
+	cmd := cloud.NewListCloudCommandForTest(s.store, func() (cloud.ListCloudsAPI, error) { return s.api, nil })
+	s.api.controllerClouds = make(map[names.CloudTag]jujucloud.Cloud)
+	s.api.controllerClouds[names.NewCloudTag("beehive")] = jujucloud.Cloud{
+		Name:      "beehive",
+		Type:      "openstack",
+		AuthTypes: []jujucloud.AuthType{"userpass", "access-key"},
+		Endpoint:  "http://myopenstack",
+		Regions: []jujucloud.Region{
+			{
+				Name:     "regionone",
+				Endpoint: "http://boston/1.0",
+			},
+		},
+	}
+
+	ctx, err := cmdtesting.RunCommand(c, cmd, "--controller", "mycontroller")
```

Comment:
probably copied from other tests, but there's no check in the fake api func that controller name of "mycontroller" is passed in